setup.cfg

```diff
@@ -37,33 +37,15 @@ select =
 
 [mypy]
 ignore_missing_imports = True
-
-[mypy-cudf._lib.*]
-ignore_errors = True
-
-[mypy-cudf._version]
-ignore_errors = True
-
-[mypy-cudf.utils.metadata.orc_column_statistics_pb2]
-ignore_errors = True
-
-[mypy-cudf.tests.*]
-ignore_errors = True
-
-[mypy-dask_cudf._version]
-ignore_errors = True
-
-[mypy-dask_cudf.tests.*]
-ignore_errors = True
-
-[mypy-custreamz._version]
-ignore_errors = True
-
-[mypy-custreamz.tests.*]
-ignore_errors = True
-
-[mypy-cudf_kafka._version]
-ignore_errors = True
-
-[mypy-cudf_kafka.tests.*]
```

Comment:
TIL we don't have any `cudf_kafka` tests. https://github.com/rapidsai/cudf/tree/branch-22.10/python/cudf_kafka/cudf_kafka